File: hackerearth/Dice.java
import java.util.Scanner;

public class Dice{
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        String input = scan.next();

        int sum = 0;
        int exception = 0;
        for(int i = 0;i < input.length();i++){
            int number = Integer.parseInt(String.valueOf(input.charAt(i)));

            if(number < 6){
                sum++;
            }

            if((i == input.length() - 1) || (i == input.length() - 2)){
                if(number == 6){
                    exception++;
                }
            }
        }

        if(exception == 2){
            System.out.println("-1");
        } else {
            System.out.println(sum);
        }
    }
}

File: hackerearth/dice.py
number = int(input())

count = 0
temp = 0
remainder = 0

length = 0
while(number != 0):
    remainder = number % 10
    temp = temp * 10 + remainder 
    number = number // 10
    length += 1

while(temp != 0):
    remainder = remainder % 10
    temp = temp // 10
    length -= 1
    if(remainder < 6):
        count += 1
    elif(remainder == 6 and length == -1):
        print("-1")
        exit()

print(count)
